target_chains/cosmwasm/tools/src/helper.ts

```diff
@@ -0,0 +1,113 @@
+import { readFileSync, existsSync, mkdirSync } from "fs";
+import { rimrafSync } from "rimraf";
+import AdmZip from "adm-zip";
+import path from "path";
+import { DownloaderHelper } from "node-downloader-helper";
+
+// We have released the compile contacts on github. If a chain needs some specific
+// feature in a contract, a version of the contract with that specific features is
+// also released. For example, "injective.zip" for injective.
+// Given a contract version (eg 1.2.0) and  zipFileNames (eg ["injective", "osmosis"])
+// this method will return an object with key = zipFileName and value = compiledCode
+export async function getContractBytesDict(
+  artifactZipFileNames: string[],
+  version: string
+) {
+  const githubArtifactsLink = `https://github.com/pyth-network/pyth-crosschain/releases/download/pyth-cosmwasm-contract-v${version}/`;
```

Comment:
i think it's a little questionable to hardcode this URL here. This ties the implementation to our git repository url, which is weird. It seems like this should be configurable and passed in. I don't feel strongly about this comment in this instance though -- no one else is going to run this -- so feel free to ignore.

I think it's also worthwhile clarifying exactly what expectations this function has on the zip files (e.g., it needs the wasm artifact inside to have a specific name, etc)

Reply:
Clarified the comment.